.idea/compiler.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```

Comment:
Coś poszło nie tak z tym pull requestem, bo jest tutaj strasznie dużo plików, które nie są pod to zadanie
![image](https://user-images.githubusercontent.com/17989854/133377297-a12e1aa8-6e70-47a4-a843-a5a26b16752c.png)

Reply:
zle Push zrobilem, poprawie